modules/installation-osp-downloading-playbooks.adoc

```diff
@@ -32,6 +32,7 @@ $ xargs -n 1 curl -O <<< '
         https://raw.githubusercontent.com/openshift/installer/release-{product-version}/upi/openstack/down-network.yaml
         https://raw.githubusercontent.com/openshift/installer/release-{product-version}/upi/openstack/down-security-groups.yaml
         https://raw.githubusercontent.com/openshift/installer/release-{product-version}/upi/openstack/down-containers.yaml'
```

Comment:
the end of the string needs to be moved to the next line, otherwise the command will fail